lib/repo/subversion_repository.rb

```diff
@@ -201,6 +201,48 @@ def stringify_files(files)
     end
     alias download_as_string stringify_files # create alias
 
+    # Generate and write the SVN authorization file for the repo.
+    def self.__generate_authz_file
+      return true if !MarkusConfigurator.markus_config_repository_admin?
+      valid_groupings_and_members = {}
+      assignments = Assignment.all
+      assignments.each do |assignment|
+        valid_groupings = assignment.valid_groupings
+        valid_groupings.each do |gr|
+          accepted_students = gr.accepted_students
+          accepted_students = accepted_students.map { |student| student.user_name }
```

Comment:
Line is too long. [83/80]<br>Pass `&:user_name` as an argument to `map` instead of a block.